### ISSUE TITLE
For Ready-to-use req.txt for training. (Create new txt file and paste this text, Required python 3.8)

### ISSUE THREAD

**Issue by iamshreeji-copy2** [jik876/hifi-gan#155]
absl-py==2.0.0
audioread==3.0.1
cffi==1.16.0
cycler==0.12.1
decorator==5.1.1
grpcio==1.59.0
importlib-metadata==6.8.0
joblib==1.3.2
kiwisolver==1.4.5
librosa==0.7.2
llvmlite==0.31.0
Markdown==3.5
MarkupSafe==2.1.3
matplotlib==3.1.3
numba==0.48.0
numpy==1.17.4
Pillow==9.5.0
protobuf==3.20.0
pycparser==2.21
pyparsing==3.1.1
python-dateutil==2.8.2
resampy==0.2.2
scikit-learn==1.2.2
scipy==1.4.1
six==1.16.0
SoundFile==0.10.3.post1
tensorboard==2.0.0
threadpoolctl==3.2.0
torch==1.4.0
Werkzeug==3.0.0
zipp==3.17.0
.
.
.
Just paste this dependency to the requirement.txt and install it at your env. It will work. Thanks in advance.
.
If all works with your system, Remember to give the up thumb. It's free.

**Comment by iamshreeji-copy1** [comment]
name: speech_synthesis_hifigan3.8_env
channels:
  - defaults
  - https://repo.anaconda.com/pkgs/main
  - https://repo.anaconda.com/pkgs/r
dependencies:
  - _libgcc_mutex=0.1=main
  - _openmp_mutex=5.1=1_gnu
  - ca-certificates=2025.2.25=h06a4308_0
  - ld_impl_linux-64=2.40=h12ee557_0
  - libffi=3.4.4=h6a678d5_1
  - libgcc-ng=11.2.0=h1234567_1
  - libgomp=11.2.0=h1234567_1
  - libstdcxx-ng=11.2.0=h1234567_1
  - ncurses=6.4=h6a678d5_0
  - openssl=3.0.16=h5eee18b_0
  - python=3.8.20=he870216_0
  - readline=8.2=h5eee18b_0
  - setuptools=75.1.0=py38h06a4308_0
  - sqlite=3.45.3=h5eee18b_0
  - tk=8.6.14=h39e8969_0
  - wheel=0.44.0=py38h06a4308_0
  - xz=5.6.4=h5eee18b_1
  - zlib=1.2.13=h5eee18b_1
  - pip:
      - absl-py==2.2.2
      - audioread==3.0.1
      - certifi==2025.1.31
      - cffi==1.17.1
      - charset-normalizer==3.4.1
      - cycler==0.12.1
      - decorator==5.2.1
      - grpcio==1.70.0
      - idna==3.10
      - importlib-metadata==8.5.0
      - importlib-resources==6.4.5
      - joblib==1.4.2
      - kiwisolver==1.4.7
      - lazy-loader==0.4
      - librosa==0.7.2
      - llvmlite==0.31.0
      - markdown==3.7
      - markupsafe==2.1.5
      - matplotlib==3.1.3
      - msgpack==1.1.0
      - numba==0.48.0
      - numpy==1.23.5
      - packaging==24.2
      - pip==21.0
      - platformdirs==4.3.6
      - pooch==1.8.2
      - protobuf==3.20.3
      - pycparser==2.22
      - pyparsing==3.1.4
      - python-dateutil==2.9.0.post0
      - requests==2.32.3
      - resampy==0.2.2
      - scikit-learn==1.3.2
      - scipy==1.10.1
      - six==1.17.0
      - soundfile==0.13.1
      - soxr==0.3.7
      - tensorboard==2.0.0
      - threadpoolctl==3.5.0
      - torch==1.4.0
      - typing-extensions==4.13.2
      - urllib3==2.2.3
      - werkzeug==3.0.6
      - zipp==3.20.2
prefix: /home/linux/miniconda3/envs/speech_synthesis_hifigan3.8_env
